packages/nodes-base/nodes/Google/BigQuery/v2/actions/database/executeQuery.operation.ts

```diff
@@ -237,6 +292,33 @@ export async function execute(this: IExecuteFunctions): Promise<INodeExecutionDa
 				body.useLegacySql = false;
 			}
 
+			if (typeof body.queryParameters === 'object') {
+				const { namedParameters } = body.queryParameters as IQueryParameterOptions;
+
+				body.parameterMode = 'NAMED';
+
+				body.queryParameters = namedParameters.map(({ name, value }) => {
+					for (const resolvable of getResolvables(name)) {
```

Comment:
No need to resolve the expression for name and value here, they are already resolved